gettsim/benefits/wohngeld.py

```diff
@@ -205,13 +198,17 @@ def _calc_wg_income_deductions_until_2015(household, params):
     )
     workingchild = household["kind"] & (household["bruttolohn_m"] > 0)
```

Comment:
I don't really get what to check here?

Reply:
The actual ruling (as also noted in the params) requires that the child must be eligible for child benefit (hours or income-wise). If it's not, then there's no allowance on the wohngeld-relevant income.

Reply:
Okay. But do you want to check this again? Because wohngeld is one of the last functions called from `tax_transfer.py`. So use it as a pre-defined variable or check again?

Reply:
I'd simply call `kindergeld_anspruch_regel()`

Reply:
Moved all eligibility checks to `kindergeld`

Reply:
I think that is the most efficient way!